lib/pages/select_language.dart

```diff
@@ -122,24 +122,24 @@ class _SelectLanguageState extends State<SelectLanguage> {
           Get.updateLocale(Locale("$languageCode"));
         });
   }
+}
 
-  String getLanguageString(
-      {required String languageCode, required BuildContext context}) {
-    final String language;
+String getLanguageString(
+    {required String languageCode, required BuildContext context}) {
+  final String language;
 
-    switch (languageCode) {
-      case 'en':
-        language = AppLocalizations.of(context)!.english;
-        break;
-      case 'es':
-        language = AppLocalizations.of(context)!.spanish;
-        break;
-      case 'ru':
-        language = AppLocalizations.of(context)!.russian;
-        break;
-      default:
-        language = 'English';
-    }
-    return language;
+  switch (languageCode) {
```

Comment:
You can have a shorter LOC (line of code) without using more `language` variable:

```dart
switch (languageCode) {
      case 'en':
        return AppLocalizations.of(context)!.english;
        break;
      ...
      default:
        return 'English';
    }
```